Tests/demistomock/demistomock.py

```diff
@@ -741,6 +741,16 @@ def incidents(incidents=None):
         )
 
 
+def incident():
+    """Retrieves the current incident
```

Comment:
@kirbles19 please review the docstring here

it will be presented in https://xsoar.pan.dev/docs/reference/articles/demisto-class